app/src/main/java/com/example/parkingsystemjava/fragment/DateTimeReservationDialogFragment.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystemjava.fragment;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.DialogFragment;
+import com.example.parkingsystemjava.databinding.DialogFragmentDateTimeReservationBinding;
+
+public class DateTimeReservationDialogFragment extends DialogFragment {
+
+    public static DateTimeReservationDialogFragment newInstance() {
+        DateTimeReservationDialogFragment instanceDateTimeReservationDialogFragment = new DateTimeReservationDialogFragment();
+        Bundle bundle = new Bundle();
+        instanceDateTimeReservationDialogFragment.setArguments(bundle);
```

Comment:
You really need include any argument?

Reply:
I think that always needed, I deleted now and until works :D